### PR TITLE
Add scss

### PR DESCRIPTION
**Pull request by k88hudson**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -48,16 +49,19 @@
     "yamscripts": "^0.0.2"
   },
   "scripts": {
+    "bundle": "npm-run-all bundle:*",
```
Comment (k88hudson) [inline package.json]: This is what yamscripts.yml compiles to when you run `npm run yamscripts`

File: yamscripts.yml [hunk 2]
```diff
@@ -1,16 +1,24 @@
...
-# When you add or modify anything, you *MUST* run npm run yamscripts
-# to compile your changes
+# When you add or modify anything, you *MUST* run:
+#      npm run yamscripts
+# to compile your changes.
 
 scripts:
+
+# bundle: Bundle assets for production
+  bundle:
```
Comment (k88hudson) [inline yamscripts.yml]: This compiles to `npm-run-all bundle:*`

File: yamscripts.yml [hunk 3]
```diff
@@ -1,16 +1,24 @@
...
+    webpack: webpack
+    css: node-sass src/main.scss -o data/content
+
 # start: Start watching/compiling assets,
-# launch addon with jpm run
   start:
     _parallel: true
     webpack: webpack -w
+    css: =>bundle:css && =>bundle:css -- -w
```
Comment (k88hudson) [inline yamscripts.yml]: The `=>` is a short form for `npm run`

Comment (rlr) [inline yamscripts.yml]: neat!

**Comment by rlr** [comment]
changes all look good to me! r+

I was unable to test it because of this version mismatch thingy error:

```
npm ERR! Darwin 15.3.0
npm ERR! argv "/usr/local/bin/node" "/usr/local/bin/npm" "install"
npm ERR! node v4.2.1
npm ERR! npm  v2.14.7
npm ERR! code EPEERINVALID

npm ERR! peerinvalid The package webpack@2.0.6-beta does not satisfy its siblings' peerDependencies requirements!
npm ERR! peerinvalid Peer babel-loader@6.2.1 wants webpack@^1.0.0

npm ERR! Please include the following file with any support request:
npm ERR!     /Users/rlr/dev/activity-streams/npm-debug.log
```

**Comment by k88hudson** [comment]
@rlr ok, should be all good

**Pull request merged**